### ISSUE TITLE
Grey out mutually exclusive options

### ISSUE THREAD

**Issue by tormento** [staxrip/staxrip#1536]
Perhaps it's a bit complicated and requires a lot of work to go thru documentation but it would be really useful.

What about greying out mutually exclusive flags, once selected?

Such as lossless in nvenc, would make bitrate or other switches useless.

**Comment by Dendraspis** [comment]
Yeah, a lot more time to implement but also maintain something like that. Currently there is something similar when some parameters are set that other are not visible anymore, but they are necessary to a certain point.

Disabling controls have to be implemented first and also a help to tell the user why this particular control is currently not available. 🤔
